python-ufonormalizer/PKGBUILD

```diff
@@ -4,37 +4,38 @@
 pkgname=python-ufonormalizer
 _pyname=${pkgname#python-}
 pkgver=0.6.1
-pkgrel=3
-pkgdesc='A tool that will normalize XML and other data inside of a UFO'
+pkgrel=6
+pkgdesc="A tool that will normalize XML and other data inside of a UFO"
 arch=(any)
-url="https://github.com/unified-font-object/ufoNormalizer"
-license=(BSD)
-depends=(python)
-makedepends=(python-{build,installer,wheel}
-             python-setuptools-scm)
+url="https://github.com/unified-font-object/$_pyname"
+license=('BSD-3-Clause')
+depends=(python
+         python-setuptools-scm)
+makedepends=(python-{build,installer,wheel})
+checkdepends=(python-pytest)
 _archive="$_pyname-$pkgver"
-source=("https://files.pythonhosted.org/packages/source/${_pyname::1}/$_pyname/$_archive.zip")
+source=("$_archive.zip::https://files.pythonhosted.org/packages/source/u/$_pyname/$_archive.zip")
 sha256sums=('e61110e75a500083f265385b1354b578610f9542e3bbbeedb98a2a6155e4aa6c')
 
 prepare() {
-	cd "$_archive"
-	# Upstream Issue: https://github.com/unified-font-object/ufoNormalizer/issues/87
-	sed -i -e 's/ "wheel",//' pyproject.toml
+  cd "$_archive"
+  # Upstream Issue: https://github.com/unified-font-object/ufoNormalizer/issues/87
+  sed -i -e 's/ "wheel",//' pyproject.toml
 }
 
 build() {
-	cd "$_archive"
-	python -m build -wn
+  cd "$_archive"
+  python -m build --wheel --no-isolation
 }
 
 check() {
-	cd "$_archive"
-	export PYTHONPATH="$PWD/build/lib"
-	python -m unittest discover
+  cd "$_archive"
+  export PYTHONPATH="$PWD/build/lib"
+  pytest
 }
 
 package() {
-	cd "$_archive"
-	python -m installer -d "$pkgdir" dist/*.whl
-	install -Dm0644 -t "$pkgdir/usr/share/licenses/$pkgname/" LICENSE.txt
+  cd "$_archive"
+  python -m installer --destdir="$pkgdir" dist/*.whl
+  install -Dm0644 -t "$pkgdir/usr/share/licenses/$pkgname/" LICENSE.txt
```

Comment:
Please don't change the coding style just to change things. Tabs for indentation, and the short flags are my preference. And I diff packages with hundreds of others I maintain and expect to be able to grep them for things the way I usually code them, so changing it up hinders my workflow. Sorry.

(The change to calling pytest directly in this block is fine although I'm not sure it serves any purpose really.)